addons/hr_work_entry/static/src/views/work_entry_calendar/work_entry_multi_selection_buttons.js

```diff
@@ -40,7 +40,7 @@ export class WorkEntryCalendarMultiSelectionButtons extends MultiSelectionButton
         }
         props.multiCreateRecordProps.context = {
             default_is_manual: true,
-            default_employee_id: this.actionService.currentController.currentState.active_id,
+            default_employee_id: this.props.reactive.context.default_employee_id,
         }
```

Comment:
Here we are overriding the context. Is there any specific reason for that?
If not, we should pass default_is_manual, so that we don’t need to set default_employee_id again.

```js
 props.multiCreateRecordProps.context  = { ...props.multiCreateRecordProps.context, default_is_manual: true };

```